dali/python/nvidia/dali/plugin/tf.py

```diff
@@ -59,26 +62,31 @@
 
 Parameters
 ----------
-    input_datasets : tf.data.Dataset or tuple of tf.data.Dataset, optional, default = None
-        input datasets to the DALI Pipeline. When provided, user must specify the ``input_names``
-        as well to provide the mapping of input datasets to ``External Source`` nodes.
+    input_datasets : dict[str, tf.data.Dataset] or dict[str, nvidia.dali.plugin.tf.experimental.input]
+        input datasets to the DALI Pipeline. It must be provided as a dictionary mapping from
+        the names of the ``External Source`` nodes to the datasets objects intended as inputs
```

Comment:
nitpick: "intended as inputs for those nodes" seems a bit redundant

Reply:
Removed.